pkg/agent/agent.go

```diff
@@ -342,11 +353,12 @@ func (i *Initializer) setupGatewayInterface() error {
 	} else {
 		klog.V(2).Infof("Gateway port %s already exists on OVS bridge", i.hostGateway)
 	}
+
 	// Idempotent operation to set the gateway's MTU: we perform this operation regardless of
 	// whether or not the gateway interface already exists, as the desired MTU may change across
 	// restarts.
 	klog.V(4).Infof("Setting gateway interface %s MTU to %d", i.hostGateway, i.mtu)
-	i.ovsBridgeClient.SetInterfaceMTU(i.hostGateway, i.mtu)
+	_ = i.ovsBridgeClient.SetInterfaceMTU(i.hostGateway, i.mtu)
```

Comment:
Any reason to change this?

Reply:
by convention, go expect u to handle error returned by function. If choose not to, do so explicitly.